src/platforms/stlinkv3/README.md

```diff
@@ -29,15 +29,27 @@ some situations, Romtable access may also fail on STM32 device and so a debugger
 warm plug will fail. Cold plug should work with any STM32 device.
 
 ## Performance
-Stock firmware has three settings: High-performance (192M), Standard frequency (96M), Low-consumption (48M).
-SWD up to 24000 kHz, JTAG up to 21333 kHz (SPI), SWO up to 16 Mbaud, VCP 732-16000000 baud.
-Suffix 7 devices are rated for -40..+125 deg C but at 200 MHz frequency maximum.
-Other mentioned limits from DS11853 are 144M, 168M, 180M for scales 3-2-1 and overdrive on/off.
-
-This firmware always runs at 216 MHz (suffix 6). AHB clock = 216M, APB1 = 54M, APB2 = 108M.
-USART1, USART6 on APB2 (108M Pclk), others (including UART5) on APB1.
-SWO capture is implemented as UART5 Rx DMA. 3375k (4500k with OVER8) is the limit.
-13.5 Mbaud (18M with OVER8) could be derived from Sysclk, 216/N where N>=16 (OVER8: 2*216/N where 24<=N<=31).
+(UM2448) Stock firmware has three settings: High-performance (192M), Standard frequency (96M), Low-consumption (48M).
+SWD up to 24000 kHz (then 8000 kHz, 3333, ...); JTAG up to 21333 kHz (both likely via SPI5, SPI3), has a return clock pin, does not support daisy chaining of targets.
+
+SWO up to 16 Mbaud, VCP 732-16000000 baud.
+See 14.2 Baud rate computing: 192/N where 16<=N<=65535 (OVER8: 2*192/N where 24<=N<=31). Substitute 96 or 48 correspondingly.
+
+(RN0093) "On STLINK-V3 boards in high-performance mode, the minimum baud rate for Virtual COM port is 2931 bauds."
+2931*65535~=192M, or rather 192M/65535~=2929.7, which also confirms the first system frequency.
```

Comment:
is that 192M in MHz - if so, please include the full units (in all relevant places).

Reply:
Expanded to MHz or millions everywhere in changed snippets.